hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/util/EntityAvroConverter.java

```diff
@@ -93,4 +84,28 @@ public static Entity convertToAvroEntity(@Nullable org.hypertrace.entity.data.se
 
     return attributeMap;
   }
+
+  private static Optional<String> convertValueToString(org.hypertrace.entity.data.service.v1.Value value) {
+    switch (value.getTypeCase()) {
+      case STRING:
+        return Optional.of(value.getString());
+      case INT:
+        return Optional.of(String.valueOf(value.getInt()));
+      case LONG:
+        return Optional.of(String.valueOf(value.getLong()));
+      case BYTES:
+        return Optional.of(HexUtils.getHex(value.getBytes().asReadOnlyByteBuffer()));
+      case DOUBLE:
+        return Optional.of(String.valueOf(value.getDouble()));
+      case FLOAT:
+        return Optional.of(String.valueOf(value.getFloat()));
+      case TIMESTAMP:
+        return Optional.of(String.valueOf(value.getTimestamp()));
+      case BOOLEAN:
+        return Optional.of(String.valueOf(value.getBoolean()));
+      default:
```

Comment:
This seems unnecessary since you're logging in the next step anyways right?

Reply:
nm. ignore this.

Reply:
No I'm not.